src/functionaltest/java/com/ericsson/ei/frontend/pageobjects/IndexPage.java

```diff
@@ -37,6 +39,13 @@ public TestRulesPage clickTestRulesPage() throws ClientProtocolException, IOExce
 
         return testRulesPage;
     }
+    
+    public SubscriptionPage clickSubscriptionPage() throws ClientProtocolException, IOException {
+      WebElement subscriptionBtn = driver.findElement(By.id("subscriptionBtn"));
```

Comment:
Timing issue will occur if the button isnt yet loaded into the html code. You would need a "wait until presence of subscriptionBtn" i think. This is applied to every time you click something and something else has to appear on the website. It's asynchronous so you cant expect it to be there at the moment you click, i saw this in a few places more in the code.